rllib/utils/tests/test_utils.py

```diff
@@ -10,10 +10,12 @@
 from ray.rllib.utils.test_utils import check
 from ray.rllib.utils.tf_utils import (
     flatten_inputs_to_1d_tensor as flatten_tf,
+    l2_loss as tf_l2_loss,
     one_hot as one_hot_tf,
 )
 from ray.rllib.utils.torch_utils import (
     flatten_inputs_to_1d_tensor as flatten_torch,
+    l2_loss as torch_l2_loss,
```

Comment:
Just saw that we implement our own torch l2 loss, which has an mse loss https://pytorch.org/docs/master/generated/torch.nn.MSELoss.html?highlight=mse%20loss#torch.nn.MSELoss that accepts a "sum" keyword.

Reply:
what do you mean?

Reply:
Similarely to tf, we do our own math here! Not a real issue as long as the math is correct. But we had the case in another PR that we decided to better rely on Tensorbackends to do the match if they do it well!

Reply:
oh I see. yeah, I don't know why we rolled these loss functions ourselves. some historical context that I don't have I think.